### PR TITLE
Bug: Fix delegate search

### PR DESCRIPTION
**Pull request by kalgoop**
filterBy pipe of ngx-filter wasn't filtering the list, in the previous version.

Now, used ngFor with ngIf to filter according to searchQuery.username

## Types of changes

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (improve a current implementation without adding a new feature or fixing a bug)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Build (changes that affect the build system)
- [ ] Docs (documentation only changes)
- [ ] Test (adding missing tests or fixing existing tests)
- [ ] Other... Please describe:

## Checklist

- [x] I have read the [CONTRIBUTING](https://docs.ark.io/guidebook/contribution-guidelines/contributing.html) documentation
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: src/pages/delegates/delegates.html [hunk 1]
```diff
@@ -30,46 +30,40 @@
...
-                <ion-col col-2 text-center><span class="percentage">{{ delegate?.productivity | number:0 }}%</span></ion-col>
-              </ion-row>
-              <ion-row *ngIf="(activeDelegates | filterBy: searchQuery).length === 0">
-                <ion-col padding text-center>
-                  <h6>{{ 'DELEGATES_PAGE.NO_MATCHING_SEARCH' | translate }}</h6>
-                </ion-col>
-              </ion-row>
-            </ion-grid>
-          </ion-item>
+          <template ngFor let-delegate [ngForOf]="activeDelegates">
```
Comment (dated) [inline src/pages/delegates/delegates.html]:
```suggestion
          <ng-template ngFor let-delegate [ngForOf]="activeDelegates">
```

File: src/pages/delegates/delegates.html [hunk 2]
```diff
@@ -30,46 +30,40 @@
...
+                  <ion-col col-2 text-center><span class="rate">{{ delegate?.rate }}</span></ion-col>
+                  <ion-col col-8>
+                    <span class="username">{{ delegate?.username }}</span>
+                    <ion-badge item-end *ngIf="isSameDelegate(delegate?.publicKey)" color="danger">{{ 'DELEGATES_PAGE.VOTED' | translate | lowercase }}</ion-badge>
+                  </ion-col>
+                  <ion-col col-2 text-center><span class="percentage">{{ delegate?.productivity | number:0 }}%</span></ion-col>
+                </ion-row>
+              </ion-grid>
+            </ion-item>
+          </template>
```
Comment (dated) [inline src/pages/delegates/delegates.html]:
```suggestion
          </ng-template>
```

File: src/pages/delegates/delegates.html [hunk 3]
```diff
@@ -30,46 +30,40 @@
...
-                <ion-col col-2 text-center><span class="percentage">{{ delegate?.productivity | number:0 }}%</span></ion-col>
-              </ion-row>
-              <ion-row *ngIf="(activeDelegates | filterBy: searchQuery).length === 0">
-                <ion-col padding text-center>
-                  <h6>{{ 'DELEGATES_PAGE.NO_MATCHING_SEARCH' | translate }}</h6>
-                </ion-col>
-              </ion-row>
-            </ion-grid>
-          </ion-item>
+          <template ngFor let-delegate [ngForOf]="standByDelegates">
```
Comment (dated) [inline src/pages/delegates/delegates.html]:
```suggestion
          <ng-template ngFor let-delegate [ngForOf]="standByDelegates">
```

File: src/pages/delegates/delegates.html [hunk 4]
```diff
@@ -30,46 +30,40 @@
...
+                  <ion-col col-2 text-center><span class="rate">{{ delegate?.rate }}</span></ion-col>
+                  <ion-col col-8>
+                    <span class="username">{{ delegate?.username }}</span>
+                    <ion-badge item-end *ngIf="isSameDelegate(delegate?.publicKey)" color="danger">{{ 'DELEGATES_PAGE.VOTED' | translate | lowercase }}</ion-badge>
+                  </ion-col>
+                  <ion-col col-2 text-center><span class="percentage">{{ delegate?.productivity | number:0 }}%</span></ion-col>
+                </ion-row>
+              </ion-grid>
+            </ion-item>
+          </template>
```
Comment (dated) [inline src/pages/delegates/delegates.html]:
```suggestion
          </ng-template>
```

**Comment by kalgoop** [comment]
@dated I have seen 'let-' being used with <template> at a dozen of places. Can you please tell why that wasn't working with <template> ?

**Comment by dated** [comment]
hi there! the problem was that the `<template>` is deprecated in angular 5.

**Comment by kalgoop** [comment]
> hi there! the problem was that the `<template>` is deprecated in angular 5.

Oh, thanks. I had Angular 4 installed in my PC.

**Pull request merged**